agents/terminal/src/main/resources/org.eclipse.che.terminal.script.sh

```diff
@@ -159,50 +149,23 @@ eval "LOCAL_AGENT_BINARIES_URI=${LOCAL_AGENT_BINARIES_URI}"
 eval "DOWNLOAD_AGENT_BINARIES_URI=${DOWNLOAD_AGENT_BINARIES_URI}"
 eval "TARGET_AGENT_BINARIES_URI=${TARGET_AGENT_BINARIES_URI}"
 
-LOCAL_AGENT_PATH=
 if [ -f "${LOCAL_AGENT_BINARIES_URI}" ]; then
     AGENT_BINARIES_URI="file://${LOCAL_AGENT_BINARIES_URI}"
-    LOCAL_AGENT_PATH=${LOCAL_AGENT_BINARIES_URI}
 elif [ -f $(echo "${LOCAL_AGENT_BINARIES_URI}" | sed "s/-${PREFIX}//g") ]; then
     AGENT_BINARIES_URI="file://"$(echo "${LOCAL_AGENT_BINARIES_URI}" | sed "s/-${PREFIX}//g")
-    LOCAL_AGENT_PATH=$(echo "${LOCAL_AGENT_BINARIES_URI}" | sed "s/-${PREFIX}//g")
 else
+    echo "Terminal Agent will be downloaded from Workspace Master"
     AGENT_BINARIES_URI=${DOWNLOAD_AGENT_BINARIES_URI}
 fi
 
-# If file is already on the filesystem, use it
-if [ ! -z ${LOCAL_AGENT_PATH} ]; then
-  tar zxf ${LOCAL_AGENT_PATH} -C ${CHE_DIR}
-else
-  echo "Terminal Agent binary is downloaded remotely"
-  # Use curl
-  if [ ${CURL_INSTALLED} = true ]; then
-    if curl -o /dev/null --silent --head --fail $(echo ${AGENT_BINARIES_URI} | sed 's/\${PREFIX}/'${PREFIX}'/g'); then
-      curl -o $(echo ${TARGET_AGENT_BINARIES_URI} | sed 's/\${PREFIX}/'${PREFIX}'/g' | sed 's/file:\/\///g') -s $(echo ${AGENT_BINARIES_URI} | sed 's/\${PREFIX}/'${PREFIX}'/g')
-    elif curl -o /dev/null --silent --head --fail $(echo ${AGENT_BINARIES_URI} | sed 's/-\${PREFIX}//g'); then
-      curl -o $(echo ${TARGET_AGENT_BINARIES_URI} | sed 's/\${PREFIX}/'${PREFIX}'/g' | sed 's/file:\/\///g') -s $(echo ${AGENT_BINARIES_URI} | sed 's/-\${PREFIX}//g')
-    fi
-    curl -s $(echo ${TARGET_AGENT_BINARIES_URI} | sed 's/\${PREFIX}/'${PREFIX}'/g') | tar  xzf - -C ${CHE_DIR}
-  else
-    # replace https by http as wget may not be able to handle ssl
-    AGENT_BINARIES_URI=$(echo ${AGENT_BINARIES_URI} | sed 's/https/http/g')
-
-    # use wget
-    WGET_SPIDER="wget --spider"
```

Comment:
Thanks for comment. I copied/pasted it from openshift-connector branch. I'll revise all changes in installer scripts and notify you.